### ISSUE TITLE
[Bug] EPSG search field in Startup not working in 7.8.x

### ISSUE THREAD

**Issue by nilason** [OSGeo/grass#1319]
**Describe the bug**
The  EPSG search field in Startup > Define new GRASS Location doesn't work on 7.8.5, at least not for Mac ~~(not tested on other platform)~~.

Just tested on Windows 7.8.1 (!), where searching works, with pressing `enter` ↩️ after typing the search text.
This doesn't work at all on Mac.

**Screenshots**

<img width="841" alt="grass785" src="https://user-images.githubusercontent.com/14186207/107526313-03646e00-6bb8-11eb-8120-7081c106df33.png">

The equivalent function works as expected in 7.9:

<img width="702" alt="grass79" src="https://user-images.githubusercontent.com/14186207/107526418-1f680f80-6bb8-11eb-941b-5a5f8484735c.png">

**System description (please complete the following information):**

- Operating System: Mac
- GRASS GIS version 7.8.5

**Comment by cmbarton** [comment]
> **Describe the bug**
> The EPSG search field in Startup > Define new GRASS Location doesn't work on 7.8.5, at least not for Mac ~(not tested on other platform)~.
> 
> Just tested on Windows 7.8.1 (!), where searching works, with pressing `enter` ↩️ after typing the search text.
> This doesn't work at all on Mac.
> 
> **Screenshots**
> 
> <img alt="grass785" width="841" src="https://user-images.githubusercontent.com/14186207/107526313-03646e00-6bb8-11eb-8120-7081c106df33.png">
> 
> The equivalent function works as expected in 7.9:
> 
> <img alt="grass79" width="702" src="https://user-images.githubusercontent.com/14186207/107526418-1f680f80-6bb8-11eb-941b-5a5f8484735c.png">
> 
> **System description (please complete the following information):**
> 
> * Operating System: Mac
> * GRASS GIS version 7.8.5

My students claim that search is not working on Windows in 7.8.5.

On my Mac, no search works in 7.8.6 dev. But the search on the EPSG list does work in 7.9.

**Comment by nilason** [comment]
> My students claim that search is not working on Windows in 7.8.5.

What I could test, on Win w. 7.8.1, search result wasn't updated dynamically as you type (as is the case for 7.9), but you have to press enter.

**Comment by cmbarton** [comment]
I'll ask them to check further and report back

**Comment by cmbarton** [comment]
Still waiting on a Windows report, but found out that search works for EPSG (with enter) but not the choose projection or choose datum windows for a student using Linux, Xubuntu 18.04 (installs ver. 7.8.0)

**Comment by hellik** [comment]
> My students claim that search is not working on Windows in 7.8.5.

tested the Korean example with

```
GRASS Version: 7.8.5                                                            
Code revision: 2b6ab2893                                                        
Build date: 2020-12-21                                                          
Build platform: x86_64-w64-mingw32                                              
GDAL: 3.1.4                                                                     
PROJ: 6.3.2                                                                     
GEOS: 3.8.1                                                                     
SQLite: 3.29.0                                                                  
Python: 3.7.0                                                                   
wxPython: 4.0.7                                                                 
Platform: Windows-10-10.0.19041-SP0 (OSGeo4W)     

```

the search works fine.

![grafik](https://user-images.githubusercontent.com/12000843/107562510-36225c80-6be0-11eb-9977-f0cbad0e88dc.png)

**Comment by hellik** [comment]
> What I could test, on Win w. 7.8.1, search result wasn't updated dynamically as you type (as is the case for 7.9), but you have to press enter.

same in winGRASS 7.8.5, you have to press enter to get the results

**Comment by nilason** [comment]
> > What I could test, on Win w. 7.8.1, search result wasn't updated dynamically as you type (as is the case for 7.9), but you have to press enter.
> 
> same in winGRASS 7.8.5, you have to press enter to get the results

Thanks! All evidence points to Mac-only issue then.

**Comment by cmbarton** [comment]
Note that so far I am hearing that search works for EPSG but not the other windows (e.g., custom CRS) in non-Mac systems, and doesn't work at all for Mac

**Comment by cmbarton** [comment]
Just confirmation from a couple of students using Windows version 7.8.5. Searching EPSG codes works if you hit enter as @hellik noted. 

BUT, searching for projections and datums in the lists for custom CRS creation does not work.

**Comment by cmbarton** [comment]
Wonderful. Search is fixed. Thanks much. Now to fix #758
